src/extension.ts

```diff
@@ -58,12 +63,6 @@ async function initializeDependencies() {
 
   // Acquire .NET Interactive
   initializeDependency("ms-dotnettools.dotnet-interactive-vscode", "dotnet-interactive.acquire", sdkResult?.dotnetPath);
-
-  // Start OmniSharp
-  initializeDependency("ms-dotnettools.csharp", "o.restart");
```

Comment:
It would be useful to activate the C# extension to kick off dependency acquisition. However, there is no need to execute either of the commands.

Reply:
Yeah, I'm thinking we should just not do that for now.

Reply:
Okay, confirmed that this works without issue. I will merge this and publish an update, since functionally this doesn't really change things other than no longer causing this conflict.